patches/0003-Add-BoringSSL-crypto-backend.patch

```diff
@@ -38,7 +38,7 @@ index 00000000000000..7c5fbeea717618
 +// Use of this source code is governed by a BSD-style
 +// license that can be found in the LICENSE file.
 +
-+//go:build goexperiment.boringcrypto && linux && cgo && (amd64 || arm64) && !android && !msan
++//go:build goexperiment.boringcrypto && (linux || darwin) && cgo && (amd64 || arm64) && !android && !msan
```

Comment:
Upstream Boring doesn't support macOS, this isn't necessary: https://github.com/golang/go/blob/0c934b5645c3220de21a5733c60c81e46d06d4e3/src/crypto/internal/boring/boring.go#L5